src/quo2/components/avatars/group_avatar/view.cljs

```diff
@@ -7,34 +7,27 @@
             [quo2.components.avatars.group-avatar.style :as style]))
 
 (def sizes
-  {:icon      {:x-small 12
-               :small   16
-               :medium  16
-               :large   20
-               :x-large 32}
-   :container {:x-small 20
-               :small   28
-               :medium  32
-               :large   48
-               :x-large 80}})
+  {20 {:icon 12}
```

Comment:
I think we should wait for the outcome of the current discussion before merging this pr as we would need to update it's uses here anyway 👍